test/utils/pickers/describeGregorianAdapter/describeGregorianAdapter.ts

```diff
@@ -12,7 +12,7 @@ function innerGregorianDescribeAdapter<TDate extends PickerValidDate, TLocale>(
   Adapter: new (...args: any) => MuiPickersAdapter<TDate>,
   params: DescribeGregorianAdapterParams<TDate, TLocale>,
 ) {
-  const prepareAdapter = params.prepareAdapter ?? ((e) => e);
+  const prepareAdapter = params.prepareAdapter ?? ((event) => event);
```

Comment:
I think `event` here is not actually correct. Probably a better alternative would be `adapter`.